#include<iostream>
using namespace std;
#include"exception.h"

int main()
{
	test1();
	return 0;
}